Formula/cbmc.rb

```diff
@@ -0,0 +1,39 @@
+class Cbmc < Formula
+  desc "CBMC: The C Bounded Model Checker"
+  homepage "https://www.cprover.org/cbmc/"
+  url "https://github.com/diffblue/cbmc/archive/cbmc-5.12-d8598f8.tar.gz"
+  version "5.12"
+  sha256 "733a5179d16c9804be92ed3bc799ce7c3035a7e627a17223d5d5fde3206d16af"
+
+  depends_on "cmake" => :build
+  depends_on "ninja" => :build
+
+  def install
+    system "cmake",
+      "-S.",
+      "-Bbuild",
+      "-DWITH_JBMC=OFF",
+      "-DCMAKE_BUILD_TYPE=Release",
+      "-DCMAKE_C_COMPILER=/usr/bin/clang",
```

Comment:
Does this not work with `std_cmake_args`?

Reply:
It does.  It builds with the brew clang, but there are problems with the developer command line tools clang.  I've documented this in the formula.